lib/python/picongpu/plugins/plot_mpl/jupyter_widgets/requirements.txt

```diff
@@ -0,0 +1 @@
+ipywidgets
```

Comment:
do not forget to add the new requirement:
`jupyter-matplotlib`

Reply:
you are right...i think just adding `ipympl` should do the job.